Sources/generate-swift-syntax-builder/gyb_helpers/utils.py

```diff
@@ -5,8 +5,11 @@ def make_swift_node(node):
   Given a Python `Node` from `gyb_syntax_support`, return the source code for matching node definition in Swift.
   """
   spaces = 9
-  parameters = ['name: "%s"' % node.syntax_kind]
-  
+  parameters = [
+    'name: "%s"' % node.syntax_kind,
+    'nameForDiagnostics: "%s"' % node.name_for_diagnostics or 'nil'
```

Comment:
Is this being evaluated as `('nameForDiagnostics: "%s"' % node.name_for_diagnostics) or 'nil'`? They're all `"None"` rather than `nil` 😅 .

Reply:
Oh, that did actually get evaluate incorrectly and I didn’t catch it because I didn’t build `generate-swift-syntax-builder` locally.

Fixed it now.